modules/pushapk_scriptworker/manifests/jarsigner_init.pp

```diff
@@ -60,7 +60,7 @@
             }
 
             java_ks {
-                'reference_browser':
+                'reference-browser':
```

Comment:
You may want to nuke the keystore because of this change. `java_ks` is a community plugin and I don't know how it behaves. If it follows what the official puppet modules do, then it leaves `reference_browser` in the keystore.

Reply:
Good point!

Reply:
keystore = nuked :+1: